scalding-core/src/main/scala/com/twitter/scalding/typed/Sketched.scala

```diff
@@ -28,14 +28,15 @@ case class Sketched[K, V](pipe: TypedPipe[(K, V)],
   seed: Int)(implicit serialization: K => Array[Byte],
     ordering: Ordering[K])
   extends HasReducers {
+  import CMSHasherImplicits._
 
   val reducers = Some(numReducers)
 
   private lazy val murmurHash = MurmurHash128(seed)
   def hash(key: K): Long = murmurHash(serialization(key))._1
 
-  private lazy implicit val cms = CMS.monoid(eps, delta, seed)
-  lazy val sketch: TypedPipe[CMS] =
+  private lazy implicit val cms = CMS.monoid[Long](eps, delta, seed)
```

Comment:
actually it is better to fix this to be a `CMS[Array[Byte]]` and make implicit hasher following this approach:

https://github.com/twitter/algebird/blob/develop/algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala#L1113

The way we had it before is because we didn't have this ability.